.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "[javascript]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+      },
+    "[typescript]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[typescriptreact]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    }
+}
```

Comment:
Ignore this, file also it is better to format with eslint that uses prettier.

Reply:
hm ok)

Reply:
but for the frontend apps that does not work, so i have to use prettier)